functionary/core/api/v1/views/execute.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+import jsonschema
+from drf_spectacular.utils import extend_schema
+from rest_framework import permissions, status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from core.api.mixins import EnvironmentViewMixin
+from core.models.function import Function
+from core.models.package import Package
+
+from ...exceptions import BadRequest
+from ..serializers.execute import ExecuteSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class ExecuteView(APIView, EnvironmentViewMixin):
+    """View for executing a function with specified parameters."""
+
+    serializer_class = ExecuteSerializer
+    # TODO: Proper permissions
+    permission_classes = [permissions.IsAuthenticated]
+
+    @extend_schema(
+        description=("Execute a Function"),
+        request=ExecuteSerializer,
+        responses={200: None},
+        parameters=EnvironmentViewMixin.header_parameters,
+    )
+    def post(self, request: Request) -> Response:
+        """Accepts a package and function name and executes it with the passed
+        parameters. The parameters are validated against the schema stored with the
+        function prior to execution."""
+        serial = ExecuteSerializer(data=request.data)
```

Comment:
I'm a fan of absurdly long names for things, so let's not truncate in cases like this.  Just call it `serializer`.

Reply:
done